crypto/evp_extra/print.c

```diff
@@ -306,6 +308,48 @@ static int eckey_priv_print(BIO *bp, const EVP_PKEY *pkey, int indent) {
   return do_EC_KEY_print(bp, pkey->pkey.ec, indent, 2);
 }
 
+// Dilithium keys.
+
+static int do_dilithium_print(BIO *bp, const EVP_PKEY *x, int off, int ptype) {
```

Comment:
```suggestion
static int do_dilithium3_print(BIO *bp, const EVP_PKEY *x, int off, int ptype) {
```
For future clarity this function is only able to print dilithium 3 keys.

Reply:
added in [68bdaff](https://github.com/awslabs/aws-lc/pull/759/commits/68bdaff186b2d9f22eeba0d226d7081cb28f468a)